applications/community-health-check.md

```diff
@@ -0,0 +1,281 @@
+
+Once our process from data fetching to analysis will be fully automated, we will
+- create a mock database developers can use for testing data fetching and analysis
+- periodically review who has access to the db and remove unnecessary person
+
+
+### Implementation 
+To date, we have delivered six community health reports (Aragon, MetaGame, Verida, Canabis Genom DAO, LexClinic, Aave), built a [prototype of our dashboard](https://www.figma.com/proto/whU3PlyPvdoSWdhUYQW7e9/Community-Health?node-id=4428%3A3647&scaling=min-zoom&page-id=3314%3A6028&starting-point-node-id=4428%3A3647&disable-default-keyboard-nav=1&hide-ui=1) and conducted 12 user interviews. 
+
+After our first report, we have adopted a step-wise approach to integrating the network metrics, focusing on providing a clear, concise, jargon-free explanation. In addition, we have included a simulation section in our health reports to showcase what the health metric could be if the interaction pattern changes or if there is a large change in the community size. 
+
+While the development of the full dashboard is ongoing, we continue to deliver low fidelity dashboards to interested communities. This lets us interact closely with users, helping us learn more about their problems and shape onboarding material for users. Interested communities get direct access to our team of data scientist who happily investigate follow-up questions. 
+
+Early development of this project has been funded by [Aragon](https://forum.aragon.org/t/real-time-community-health-analytics), Polygon DAO, [Aave](https://noturhandle.gitbook.io/rndao/proposals/proposals/grant-proposal-aave), [MetaCartel](https://forum.metacartel.org/t/grant-request-community-health-a-project-by-rndao/2354) and Near.
+
+## Development Roadmap :nut_and_bolt:
+
+
+### Overview
+
+- **Total Estimated Duration:**  10 weeks
+- **Full-Time Equivalent (FTE):**  6
+- **Total Costs:**  15 000 USDC
+
+We are following a grant-matching processes to spread the risk among different communities. 
+
+### Milestone 1 - Defining metrics
+
+- **Estimated duration:** 4 weeks
+- **FTE:**  1
+- **Costs:** 5000 USDC
+
+Research social network metrics that are viable to measure the health of a community on Twitter, and how they integrate with our existing community health framework. This will build on our existing framework, extending it to metrics that have been tested and used in Twitter communities. For example, we will look into approaches to build the network, virality and clustering/fragmentation of very large online communities. 
+
+Note: We have done an evaluation to decide wheter Reddit or Twitter would be a better option. Given other requests we have received, we have decided that building an integration and health metrics for Twitter is a more viable path for us.
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 0a. | License | The results will be published open-access (no paywall or copyright). |
```

Comment:
Could you specify the license you're planning to use?

Reply:
@katerinabc this is still not specified. Please choose one of the [supported licenses](https://github.com/w3f/Grants-Program/#guidelines):

> We prefer Apache 2.0, but GPLv3, MIT or Unlicense are also acceptable.

If you're undecided, you can specify multiple ones and add a note, that you will pick one of them

Reply:
Apologies. I worked for a long time in academia and there it's just a given that every written document can be re-used by others just with referencing. I'm gonna add multiple ones and decide later.